deployment/provider-azure-installer-windows.yaml

```diff
@@ -21,27 +21,39 @@ spec:
         app: csi-secrets-store-provider-azure
     spec:
       serviceAccountName: csi-secrets-store-provider-azure
+      hostNetwork: true
```

Comment:
For these yaml changes, we should wait until after this https://github.com/Azure/secrets-store-csi-driver-provider-azure/pull/215 then move these to manifest_staging